hack/make/release-deb

```diff
@@ -98,6 +98,11 @@ if [ ! -f "$APTDIR/conf/docker-engine-release.conf" ]; then
 	EOF
 fi
 
+# build the right directory structure, needed for apt-ftparchive
+for arch in "${arches[@]}"; do
```

Comment:
component comes from above, but you are right for codename, I'll move. thanks for catching that.